packages/react-components/style/output-components-style.css

```diff
@@ -10,6 +10,9 @@
     display: grid;
     grid-template-rows: 25px 1fr;
 }
+.widget-handle :hover {
+    background-color: var(--theia-selection-background);
```

Comment:
Please change the colour to something very subtle (ex. slightly lighten the bar colour) like in the example in the issue description #240. Here's the information priority that should be respected in this case:
1. **What action is possible:** The user should first notice that the cursor changes to a cross with arrows (this tells them they can click to move the panel). The cursor change is the most important piece of information.
2. **Where the action is possible:** The reason the title bar changes colour is just to show the the user the full area where they can hover to move the panel. It is secondary information so the colour change should be subtle. Right now the bright blue means the user is unlikely to notice the cursor change.

Reply:
Alright, I understand. Thank you for your comment.  changes are coming soon

Reply:
Thanks for the update. I like the new colour you chose!